content/swift/concepts/sets/terms/insert.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '.insert()'
+Description: 'Adds an element at a specified index.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`insert()`** method adds or inserts a given element in the set if it is not already present because set is a collection of unique elements.
```

Comment:
```suggestion
The **`insert()`** method adds or inserts a given element in the set if it is not already present since a set must contain unique elements.
```